feat: Add maintainers guide section (#205)
open-sauced/docs: docs/maintainers/maintainers-guide.md

---
id: maintainers-guide-to-open-sauced

> title: 'Maintainers Guide to OpenSauced'

Reviewer: The titles in some files accross the repo have single quotes, others have double quotes, and several have no quote.
Which one will we use for consistency?

> sidebar_label: Maintainers Guide to OpenSauced

Reviewer: The `sidebar_label` in some files have single or double quotes (e.g., in the `insights.md` file) and some don't.
Which one should we keep for consistency?

Author: Remove them.

keywords:
  - maintainers
  - guides
---

## Introduction to OpenSauced for Maintainers
OpenSauced is a platform dedicated to empowering maintainers and redefining open source contributions. We offer a GitHub-powered dashboard for tracking contributions and gaining insights into your open source projects. You can customize your Insights pages and lists with the repositories you choose and with the contributors you want to see. 

> You can connect with your team and contributors through our platform and invite them to join OpenSauced. We also offer a GitHub browser extension that allows you to view your contributors' OpenSauced profiles and invite them to join OpenSauced with a single click.

Reviewer 2: @adiati98, would'nt it be a Chrome browser extension though? :)

To get started, you will need to create an account on OpenSauced. You can do this by visiting [app.opensauced.pizza](https://app.opensauced.pizza) and clicking the "Connect with GitHub" button. You will be prompted to sign in with your GitHub account. Once you have signed in, you will be redirected to the OpenSauced dashboard.

## Insights: Connecting Your Repositories
OpenSauced Insights provides maintainers with a visual representation of their project's health and contributor engagement. It offers real-time data and analytics to understand contribution patterns, identify key contributors, and measure the project's growth. Insights can help you to assess team activity, understand when bottlenecks occur, identify where resource allocation is needed, plan and identify key contributors, understand contributor churn, or spot emerging trends.

Insights features allow you to:
- track the contributions of a specific team, a specific repository, or group of repositories
- view the number of pull requests that are open, closed, in draft, and merged
- view the average number of days it takes for a pull request to be merged (PR velocity)
- identify the top contributors to your project

> - identify the current activity level and trends in activity of your contributors

Reviewer: Do we want to keep the first letters in the list lower case or capitalize them?

Author: lower case, since they aren't a complete sentence

#### Contributors
The contributors tab allows you to view more detailed information on each of the contributors, including Activity, Repositorites, the date of their last contribution, time zone, and number of contributions. 

To learn more about your contributors, you can select them and add them to a list. 

![contributors insight list](../../static/img/contributors-insights.png)

There are two ways to add contributors to a list:
1. Create a new list with selected contributors.
2. Add selected contributors to an existing list.

## Lists: Connecting with Contributors
OpenSauced Lists feature allows you to categorize, monitor, and analyze various groups of contributors. You can use lists to track the contributions of a specific team, a specific repository, or group of repositories. This feature offers granular insights into each contributor's activities and contributions, helping maintainers to track individual contribution histories, identify active engagement, recognize new or alumni contributors, and compare the performance of different contributors.

The Lists features allow you to:
- track the commits and PRs of the contributors on your list

> - filter your contributors by their activity level, including most active, new, and alumni

Author: Yes, that should be updated in the next release. We're replacing everything that says churned with Alumni.

> - filter your contributors by their activity level, including most active, new, and alumni

Reviewer: I changed some to "Churned" because I follow the existing app.
Do we want to use "Alumni" from now on?
If so, I'll recheck and fix them.

> - filter your contributors by their activity level, including most active, new, and alumni

Author: Yes, please use Alumni. The only time to keep churned is in the section that describes Alumni.

> - view the the repositories that your contributors have contributed to, the languages they've used for their contributions, and their time zones.

Author: We're going to be making some updates to this, so I clarified it in a way that reflects the upcoming changes. And thanks for pointing this out bc it started a good discussion.

You can filter your list by all contributors, active contributors, new contributors, and alumni contributors.

![list graph](../../static/img/list-graph.png)

### Contributors
The contributors view, gives you the list of the contributors and includes information like activity level, last repository contributed to, the date of their last contribution, time zone, programming languages used, and number of contributions.

 To learn more about these features, see [Understanding Lists Data](#understanding-lists-data).

## Highlights: Connecting and Recognizing Contributions
Leveraging OpenSauced's Highlights feature is a great way for maintainers to recognize and showcase the valuable contributions made to your open source projects, to curate and share key achievements, contributions, and milestones, and bring visibility to the hard work of your community. 

Highlights can be used to:
- publicize issues on your repository that you'd like to match with contributors
- share Dev.to blog posts you've written
- celebrate PRs to your project

### Creating a New Highlight
There are two ways to create a new highlight:

> 1. On the [main feed](https://app.opensauced.pizza/feed), click the input that says "Post a highlight to show your work!"

Reviewer: For someone who visit OpenSauced for the first time, they might click the link to go to the feed. But probably the next time they visit, they won't find any "feed" word anywhere. Should we do this instead?

```suggestion
1. On the top bar, click the "Highlights" tab, then click the input that says "Post a highlight to show your work!"
```

2. From your profile, click the input that says "Post a highlight to show your work!"

After you've clicked the input, you'll see a form that allows you to create a new highlight.

There are two ways to create a highlight:
1. Paste the URL to your PR, Issue, or Dev.to blog post (shown in blue below).
2. Choose from a list of suggestions (shown in orange below).

![highlight suggestions](../../static/img/highlights.png)

If you're adding a blog post, you'll need to associate it to a repository by clicking the "Add a repo" button.

Finally, it's time to add the details of your highlight. You can add your own description or use the auto-summarize function to generate a description of.

![gif of creating a highlight](../../static/gif/highlight.gif)

### Writing an Effective Highlight
Writing an effective highlight is key to attracting contributors to your project. Here are some tips to help you write an effective highlight:

#### Issues

> 1. **Focus on the Problem Solved**: Begin by succinctly describing the problem or challenge the issue addresses. This sets the context for why it was important.

Reviewer: Should we do lower case for the bold ones? Or do we want to treat them like a title?

```suggestion
1. **Focus on the problem solved**: Begin by succinctly describing the problem or challenge the issue addresses. This sets the context for why it was important.
```

Author: I was treating it like a title, but I'm fine with whatever as long as we stay consistent.

Reviewer: If it is a common `ol` — for example, like instruction steps, etc. — I'd say let's do lower case.

If we want to make it a title, then we apply heading like this:

```markdown
##### 1. Focus on the Problem Solved

Begin by succinctly describing the problem or challenge the issue addresses. This sets the context for why it was important.
```

Reviewer: Oh, and I changed the word "succinctly" to "briefly" because I think it's easier to understand for non-native English speaker (I had to look up the meaning of it 😅)

Author: Thanks! That's really helpful!

---
id: maintainers-guide-to-open-sauced
title: 'Maintainers Guide to OpenSauced'
sidebar_label: Maintainers Guide to OpenSauced
keywords:
  - maintainers
  - guides
---

## Introduction to OpenSauced for Maintainers

OpenSauced is a platform dedicated to empowering maintainers and redefining open source contributions. We offer a GitHub-powered dashboard for tracking contributions and gaining insights into your open source projects. You can customize your Insights Pages and Lists with the repositories you choose and the contributors you want to see.

> You can connect with your team and contributors through our platform and invite them to join OpenSauced. We also offer a [GitHub browser extension](../tools/chrome-extension/introduction-to-the-chrome-extension.md) that allows you to view your contributors' OpenSauced profiles and invite them to join OpenSauced with a single click.

Reviewer: @BekahHW the link text here, should it be "GitHub browser extension" or "OpenSauced Chrome extension" or "GitHub intergrated OpenSauced Chrome extension"?

Author: I actually removed the mention. I thought it was more distracting there. It might be useful to have a second guide for OpenSauced Tools for Maintainers, but we don't need that yet

Reviewer: That makes sense! 😄

Reviewer 2: I second this

### Creating a New Highlight

There are two ways to create a new highlight:

1. After clicking the [Highlights tab](https://app.opensauced.pizza/feed) in the top navigation, click the input that says "Post a highlight to show your work!"

2. From your profile, click the input that says "Post a highlight to show your work!"

After you've clicked the input, you'll see a form that allows you to create a new highlight.

There are two ways to create a highlight:

1. Paste the URL to your PR, Issue, or Dev.to blog post (shown in blue below).
2. Choose from a list of suggestions (shown in orange below).

![highlight suggestions](../../static/img/highlights.png)

If you're adding a blog post, you'll need to associate it to a repository by clicking the "Add a repo" button.

Finally, it's time to add the details of your highlight. You can add a description yourself or use the auto-summarize function to generate a description.

![gif of creating a highlight](../../static/gif/highlight.gif)

### Writing an Effective Highlight

Writing an effective highlight is key to attracting contributors to your project. Here are some tips to help you write an effective highlight:

> #### Issues

Reviewer: Let's make this consistent with the rest of the sections.

```suggestion
#### Highlighting Issues
```
